### PR TITLE
Add option to pause Near contracts

### PR DESCRIPTION
**Pull request by mfornet**
Test plan
=======
- (TODO): Near contracts are compiled deterministically
- (TODO): Unit test that stress pause conditions
- e2e pass

### CODE REVIEW

File: contracts/near/eth-client/src/lib.rs [hunk 1]
```diff
@@ -58,6 +59,8 @@ pub struct HeaderInfo {
     pub number: u64,
 }
```
Comment (sept-en) [inline contracts/near/eth-client/src/lib.rs:61]: I suggest is that we either add another flag that makes it clear how to unpause everything (i.e. `const UNPAUSE_ALL: u128 = 0;` as we do similar on the Solidity side) or add one more method to the trait which - `setAllUnpaused()`.

File: contracts/near/admin-controlled/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+use near_sdk::env;
+
+pub type Mask = u128;
```
Comment (sept-en) [inline contracts/near/admin-controlled/src/lib.rs:3]: Is there any reason to use `u128` for the mask? I think even if we extend it one day to manage any functionality in contracts (not only plausibility) is still too huge. IMO the `u8` is enough for this, though we can extend it to `u16` or `u32` (for future functionality).

File: contracts/near/eth-client/src/lib.rs [hunk 3]
```diff
@@ -58,6 +59,8 @@ pub struct HeaderInfo {
...
 
+const PAUSE_ADD_BLOCK_HEADER: u128 = 1;
```
Comment (djsatok) [inline contracts/near/eth-client/src/lib.rs:62]: `u128` -> `Mask`

File: contracts/near/eth-prover/src/lib.rs [hunk 4]
```diff
@@ -45,6 +47,8 @@ impl Default for EthProver {
     }
 }
 
+const PAUSE_VERIFY: u128 = 1;
```
Comment (djsatok) [inline contracts/near/eth-prover/src/lib.rs:50]: `u128` -> `Mask`

**Review by djsatok — CHANGES_REQUESTED** [review]

**Comment by djsatok** [comment]
Closes #509

File: contracts/near/eth-client/src/lib.rs (hunk 1 above)
Comment (mfornet) [reply contracts/near/eth-client/src/lib.rs:61]: This code should probably live on the service interacting with the contract, but not in the contract itself give that it is not used.

File: contracts/near/admin-controlled/src/lib.rs (hunk 2 above)
Comment (mfornet) [reply contracts/near/admin-controlled/src/lib.rs:3]: It is better to be on the safe side. Notice this is generic interface that can be implemented for any contract.

**Pull request merged**